en/components/expansion_panel.md

```diff
@@ -0,0 +1,281 @@
+    <iframe id="expansion-sample-1-sample" frameborder="0" seamless width="100%" height="100%" src="{environment:demosBaseUrl}/expansion-sample-1" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="expansion-sample-1-sample" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+### Binding to properties and events
+
+We can add some logic to our component to make it show/hide the description depending on the state of the panel. We can do this by binding the description to the controls `collapsed` property:
+
+```typescript
+// in expansion-panel.component.ts
+import { IgxExpansionPanelComponent } from 'igniteui-angular';
+@Component({
+    ...
+})
+export class ExpansionPanelComponent {
+    @ViewChild(IgxExpansionPanelComponent, {read: IgxExpansionPanelComponent})
+    public panel: IgxExpansionPanelComponent;
+    public readMore: string = 'https://en.wikipedia.org/wiki/Hummingbird';
+}
+```
+
+```html
+<!-- in expansion-component.component.html -->
+...
+    <igx-expansion-panel-header>
+        ...
+        <igx-expansion-panel-description *ngIf="panel.collapsed">
+            One of the smallest birds
+        </igx-expansion-panel-description>
+    </igx-expansion-panel-header>
+```
+
+This way, the description will only be display when the component is collapsed.
+
+If we want to add more complex functionality depending on the component's state, we could also bind to the event emitters.
+```typescript
+// in expansion-panel.component.ts
+
+export class ExpansionPanelComponent {
+    ...
+    public handleExpansion(args: {event: Event}): void {
+        ... // Some functionality
+    }
+}
+```
+```html
+<!-- in expansion-component.component.html -->
+    <igx-expansion-panel (onExpanded)="handleExpansion($event)">
+
+    </igx-expansion-panel>
+```
+
+Below we have the results:
+<div class="sample-container loading" style="height: 200px;">
+    <iframe id="expansion-sample-2-sample" frameborder="0" seamless width="100%" height="100%" src="{environment:demosBaseUrl}/expansion-sample-2" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="expansion-sample-2-sample" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+### Styling the header
+The `IgxExpansionPanelComponent` allows for easy customization of the header.
+The default icon for the toggle state of the control can be templated. The position of this icon is also configurable - either the start or end of the header. It can also be hidden.
+
+Configuring the position of the header icon can be done through the `iconPosition` input on the `igx-expansion-panel-header`. The posible options for the icon position are **left**, **right** and **none**. Let's configure our components button to go on the *right* side.
```

Comment:
Maybe reword the last sentence to - The next code sample demonstrates how to configure the component's button to go on the *right* side